cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package encryption handles assigning encryption providers to each group resource storage.
+package encryption
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/server/options/encryptionconfig"
+	"k8s.io/apiserver/pkg/storage/value"
+	"k8s.io/apiserver/pkg/storage/value/encrypt/kms"
+	"k8s.io/kubernetes/pkg/cloudprovider"
+	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
+)
+
+// GetTransformerOverrides creates a map of transformer overrides, one for each group resource which was
+// specified in the configuration file configFilePath. cloudProvider would be used to connect to a cloud
+// when using Cloud KMS services.
```

Comment:
I used KMS service explicitly to differentiate from the KMS transformer. KMS alone might get confusing to a future reader, and we have a struct named `kms.Service` which would give context to the reader.